go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/greenplum-db/gp-common-go-libs
 
-go 1.19
+go 1.21
```

Comment:
Since the patch version is not being used here, does it pick up the latest patch version available on the machine?

Reply:
I think this is just a minimum version requirement. The 1.19 version was working on my machine even though I had 1.21 actually installed. I bumped it because I believe the new errors.Join functionality I started using was introduced in go 1.20 or 1.21. We already use go 1.21 in some other projects so I figured we could just match that version.